Makefile

```diff
@@ -32,6 +32,7 @@ TESTDATA := $(TOP_LEVEL)/test/data
 OS ?= linux
 ARCH ?= amd64
 BENCH_OUTPUT ?= stdout
+ALL_EXTENSIONS = apikey,lint,metrics,mgmt,scrub,search,sync,ui,userprefs
```

Comment:
Add `debug` to the list. That is a valid label too.

Reply:
Added.